src/fernet/core.clj

```diff
@@ -85,8 +85,19 @@
     (catch Exception e
       (invalid-token))))
 
+(defn decrypt [key token & options]
+  "Decrypt the token and return the contents as a byte-array"
+  (apply decrypt-token key token options))
+
+(defn decrypt-to-string
+  "Decrypt the token and return the message as a string"
+  [key token & options]
+  (String. (apply decrypt-token key token options)))
+
 (defn encrypt [key message]
```

Comment:
It would really help to understand this if it had type hints, even if it's only a compiler optimization (that goes for the other signatures too)